src/any_llm/types/messages.py

```
@@ -53,15 +90,67 @@ class ThinkingBlock(AnthropicThinkingBlock):
     signature: str = ""
 
 
-ContentBlock = TextBlock | ToolUseBlock | ThinkingBlock
+CompactionBlock = BetaCompactionBlock
+CompactionDelta = BetaCompactionContentBlockDelta
+
 
-MessageContentBlock = ContentBlock
+def _normalize_thinking_block(value: Any) -> Any:
```

Ruff (0.16.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)